flipperdevices/flipperzero-firmware
# [FL-1909] Desktop refactoring

# What's new

- Desktop locked view: fix hint timeout logic
- Switch HwMismatch custom view to popup

# Verification 

- Compile and upload
- Check hint when device is locked
- Check hw mismatch screen

# Checklist (do not modify)
